Ginger/GingerCoreNET/Run/RunListenerLib/LiteDBRepository.cs

```diff
@@ -95,7 +95,7 @@ public override object SetReportAction(GingerCore.Actions.Act action, Context co
                     {
                         if (File.Exists(completeSSPath))
                         {
-                            File.Delete(completeSSPath);
+                            continue;
```

Comment:
why delete was removed, were we deleting the same SS or it was for older SS?

Reply:
yes, we were deleting the same SS